CONTRIBUTING.md

```diff
@@ -81,9 +81,11 @@ overrides unless the overriding function does something different (as far as
 the caller is concerned) than the overridden function. Ddoc comment blocks
 are often overkill for nested functions and function literals; use ordinary
 comments for those. Follow the [D Style](https://dlang.org/dstyle.html#phobos_documentation)
-for for comment blocks. 
```

Comment:
removing the trailing white was a good move but I should also remove the double "for"